.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,8 @@
 <!-- What's in this pull request? -->
 Replace this paragraph with a description of your changes and rationale. Provide links to external references/discussions if appropriate.
 
-<!-- If this pull request resolves any bugs in the Swift bug tracker, provide a link: -->
-Resolves SR-NNNN.
+<!-- If this pull request resolves any GitHub issues, mention them: -->
+Resolves #NNNNN, fix apple/llvm-project#MMMMM.
```

Comment:
```suggestion
Resolves #NNNNN.
```

What is the main reason for including `fix apple/llvm-project#MMMMM.`

Reply:
I specifically used llvm-project as an example of closing an issue on an external repo, whereas you are only showing closing issues on this repo, as mentioned in the resolved conversation above.

Reply:
This can be confusing, are you ok with dropping the external repo?

Reply:
If we decide not to mention an external repo, which IMO is a legit case, since the Swift project is an umbrella, let us at least show how to link multiple issues to a PR. This was my original motivation for the comma-separated list.

Reply:
I think it's worth keeping and only added the second issue on Anthony's suggestion, but can remove it if you think it's not worth it.

Reply:
How about: 

```suggestion
Resolves:
* #NNNNN
* example/repository#MMMMM
```

Reply:
GitHub will not pick this up, you need to [repeat one of the supported keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) for every issue you want to link.

Reply:
That's good point.

Reply:
FYI I’m OK with using a mock repo for the second issue as in Mishal’s example.

Reply:
I think it's better to show an example of the most likely external repo that this compiler repo will link to, but can make it generic if you guys want.

Reply:
@shahmishal, is this pull okay now? Please approve or let me know if any more tweaks are needed.